### DISCUSSION TITLE
Kaocha interfere with refreshing of namespaces

### DISCUSSION THREAD

**Discussion by nbardiuk** [Olical/conjure#359] [General]
Hey, just wanted to share a workaround for Conjure users who run tests with kaocha and have problems with refreshing namespaces.

Steps to reproduce:
* start clojure repl
* call `refresh_all` - all namespaces are refreshed
* call `run_all_tests` - all tests are executed
* modify non-test file
* call `refresh` - nothing is refreshed, [PROBLEM] expected the modified namespace to be refreshed
* call `refresh_all` - only test namespaces are refreshed, [PROBLEM] expected all namespaces to be refreshed

Turns out this is a known issue https://github.com/lambdaisland/kaocha/issues/161 and they suggest a workaround as [Disabling Automatic Classpath Handling](https://github.com/lambdaisland/kaocha/blob/8f18babb732b21e7fb2231e44be4d972c7ab22bc/doc/03_configuration.md#disabling-automatic-classpath-handling) 

In conjure configuration we can set the flag by overriding call suffix
```vim
let g:conjure#client#clojure#nrepl#test#call_suffix = "{:kaocha/color? false :tests [{:kaocha.testable/skip-add-classpath? true}]}"
```

**Comment by Olical** [comment]
This is brilliant, thank you for finding this and putting it together. It's the first step towards a mitigation from Conjure's side too.
